content/en/docs/tasks/administer-cluster/sysctl-cluster.md

```diff
@@ -105,20 +105,25 @@ manually by the cluster admin, either by means of the underlying Linux
 distribution of the nodes (e.g. via `/etc/sysctls.conf`) or using a DaemonSet
 with privileged containers.
 
-The sysctl feature is an alpha API. Therefore, sysctls are set using annotations
+The sysctl feature is a beta API. The sysctls are set through pod security context
```

Comment:
Can you use the feature shortcode for this instead of just having it in a paragraph? Syntax is like:

```
{{< feature-state for_k8s_version="v1.11" state="beta" >}}
```

This is lots easier to maintain than having to hunt through the Markdown files for mentions of "alpha", "beta" etc in the flow of the text.